tests/Unit/Entity/Activity/PriceOptionTest.php

```diff
@@ -184,8 +185,11 @@ public function testSetActivity(): void
 
     public function test__toString(): void
     {
-        /* @todo This test is incomplete. */
-        $this->markTestIncomplete();
+        $expected = 'free €0.00';
+        $priceOption = new PriceOption();
+        $priceOption->setName('free');
+        $priceOption->setPrice(0);
+        $this->assertSame($expected, $priceOption->__toString());
```

Comment:
Given the fact that __toString() is a magic method, I feel that an assert using strval would be better, eg.
```php
$this->assertSame($expected, strval($priceOption));
```

Reply:
consider it done!